code/modules/mining/coins.dm

```diff
@@ -44,6 +44,28 @@
 	name = "platinum coin"
 	icon_state = "coin_adamantine"
 
+/obj/item/weapon/coin/erp
+	name = "erp token"
+	icon_state = "erp_token"
+	desc = "Put this in her slot to redeem your erp token. Good for 1 deciseconds of erp."
+
+/obj/item/weapon/coin/erp/attack(/mob/living/carbon/human/target, /mob/living/carbon/human/user, var/target_zone)
+	if(!istype(target) || !istype(user) || target_zone != "groin")
+		return
+	else
+		user.visible_message(\
+			"[user] inserts the [src] into [target]'s slot...",\
+			"You insert the [src] into [target]'s slot...",\
+			"You hear the distinct sound of a coin going into a vending machine..."\
```

Comment:
Tokens and coins have a distinct sound, and this would be apparent to all except perhaps IPCs. Perhaps a check for IPCs who receive this message, and then everyone else who receives the proper message? Would go along way for IPC nerfs that are sorely needed.